src/internal/storage/server.go

```diff
@@ -312,13 +312,15 @@ func (s *Server) RenewFileset(ctx context.Context, request *storage.RenewFileset
 	if err != nil {
 		return nil, err
 	}
-	ttl := time.Duration(request.TtlSeconds) * time.Second
-	if ttl < time.Second {
-		return nil, errors.Errorf("ttl (%d) must be at least one second", ttl)
-	}
-	if ttl > maxTTL {
-		return nil, errors.Errorf("ttl (%d) exceeds max ttl (%d)", ttl, maxTTL)
+	var ttl time.Duration
```

Comment:
Nits, ttl could just be declared where it is set and I don't think the comment is really necessary. The implicit structure should just be validate then process with the expectation that the inputs are valid at that point.